src/koa-npm-resolution.ts

```diff
@@ -0,0 +1,32 @@
+import * as Koa from 'koa';
```

Comment:
Could this file be combined with `koa-esm-specifier-transform.ts`? Both have a `middleware`, can there just be one, or is there a good reason they are split?

Reply:
The `koa-esm-specifier-transform.ts` is written to allow you to provide your own resolver-- My intent is to extract it and refactor the `koa-import-map` middleware to use it.

Reply:
Side note: I just renamed this to `koa-module-specifier-transform.ts`